examples/regression/3-scientific-articles/plot_gibbs2023_simulations.py

```diff
@@ -0,0 +1,369 @@
+    degree = 4
+
+    model = Pipeline(
+        [
+            ("poly", PolynomialFeatures(degree=degree)),
+            ("linear", LinearRegression())
+        ]
+    )
+    return model
+
+###############################################################################
+# 2. Generate and present data
+# -----------------------------------------------------------------------------
+
+
+def generate_data(n_train=2000, n_calib=2000, n_test=500):
+    def f(x):
+        ax = 0*x
+        for i in range(len(x)):
+            ax[i] = (np.random.poisson(np.sin(x[i])**2 + 0.1)
+                     + 0.03*x[i]*np.random.randn(1))
+            ax[i] += 25*(np.random.uniform(0, 1, 1) < 0.01)*np.random.randn(1)
+        return ax.astype(np.float32)
+
+    # training features
+    X_train = np.random.uniform(0, 5.0, size=n_train).astype(np.float32)
+    X_calib = np.random.uniform(0, 5.0, size=n_calib).astype(np.float32)
+    X_test = np.random.uniform(0, 5.0, size=n_test).astype(np.float32)
+
+    # generate labels
+    y_train = f(X_train)
+    y_calib = f(X_calib)
+    y_test = f(X_test)
+
+    # reshape the features
+    X_train = X_train.reshape(-1, 1)
+    X_calib = X_calib.reshape(-1, 1)
+    X_test = X_test.reshape(-1, 1)
+
+    return X_train, y_train, X_calib, y_calib, X_test, y_test
+
+
+X_train, y_train, X_calib, y_calib, X_test, y_test = generate_data()
+
+fig = plt.figure(figsize=(12, 5))
+ax1 = fig.add_subplot(1, 2, 1)
+ax1.scatter(X_train[:, 0], y_train, s=1.5, alpha=0.6, label="Train Data")
+ax1.set_xlabel("X")
+ax1.set_ylabel("Y")
+ax1.set_title("Train Data")
+ax1.legend()
+
+ax2 = fig.add_subplot(1, 2, 2)
+ax2.scatter(X_train[:, 0], y_train, s=1.5, alpha=0.6, label="Train Data")
+ax2.set_ylim([-2, 6])
+ax2.set_xlabel("X")
+ax2.set_ylabel("Y")
+ax2.set_title("Zoom")
+ax2.legend()
+
+plt.show()
+
+##############################################################################
+# 3. Prepare model and show predictions
+# -----------------------------------------------------------------------------
+
+model = init_model()
+
+model.fit(X_train, y_train)
+
+sort_order = np.argsort(X_test[:, 0])
+x_test_s = X_test[sort_order]
+y_pred_s = model.predict(x_test_s)
+
+plt.figure(figsize=(6, 5))
+plt.scatter(X_test[:, 0], y_test, s=1.5, alpha=0.6, label="Test Data")
+plt.plot(x_test_s, y_pred_s, "-k", label="Prediction")
+plt.ylim([-2, 6])
+plt.xlabel("X")
+plt.ylabel("Y")
+plt.title("Test Data (Zoom)")
+plt.legend()
+plt.show()
+
+
+##############################################################################
+# 4. Prepare Experiments
+# -----------------------------------------------------------------------------
+# In this experiment, we will use the
+# :class:`~mapie.regression.MapieRegressor` and
+# :class:`~mapie.regression.MapieCCPRegressor` to compute prediction intervals
+# with the basic Split CP method and the paper CCP method.
+# The coverages will be computed on 500 different dataset generation, to have
```

Comment:
on 50* no?

Reply:
Note that you can blame the documentation restrictions!

Reply:
You are write, I forgot to change it in the text. Done.